docs/variants/novacustom_ns5x_adl/releases.md

```diff
@@ -13,6 +13,81 @@ For details about our release process please read
 
 </center>
 
+## v1.7.2 - 2023-12-22
```

Comment:
We need to adjust the release date (likely to 2023-12-28). However, we will correct it before merging, so there's no need to fix that today.

Reply:
Release date adjusted to today